packages/react/src/SelectPanel/SelectPanelMessage.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Text from '../Text'
+import Octicon from '../Octicon'
+import {AlertIcon} from '@primer/octicons-react'
+import classes from './SelectPanel.module.css'
+import {clsx} from 'clsx'
+
+export type SelectPanelMessageProps = {
+  children: React.ReactNode
+  title: string
+  variant: 'empty' | 'noResults' | 'error' | 'warning'
```

Comment:
I'm not sure what our naming conventions are for this but was surprised to see pascalCase, is that what we normally do? I would have expected:

```suggestion
  variant: 'empty' | 'no-results' | 'error' | 'warning'
```

Reply:
done in 7e19c26df!